datashader/composite.py

```diff
@@ -46,7 +47,7 @@ def combine_scaled(r, g, b, a):
 
 
 def operator(f):
```

Comment:
```suggestion
def img_operator(f):
```
?

Reply:
I'm not sure about this one. If anything, I would like to remove the RGB spreading support entirely so `operator` would stay but only apply to arrays, not RGB.

Reply:
Changing the semantics but keeping the same name seems more confusing than changing the name and keeping the same semantics.  I guess just leave it as-is and someday there will only be arr_operator.